### DISCUSSION TITLE
Brainsight SimNIBS integration e-field modelling

### DISCUSSION THREAD

**Discussion by bcwbc** [simnibs/simnibs#181] [8. Simulation workflows]
Hello,
 
  I am using SimNIBS for some e-field modelling and was looking for a little bit of help. I am using the Brainsight/SimNIBS integration for modelling the e-field but was looking to do some ROI analysis after the fact. Is this possible to do using the files produced from running the simulations on Brainsight? It appears running the simulations in Brainsight produce slightly different information than running in the SimNIBS GUI. If running the simulation in the GUI is recommended, I am unsure how to mimic an identical stimulation that was done already on a participant in Brainsight in the SimNIBS GUI and then analyze the e-field in a ROI.
 
  I apologize if I am missing something, I am relatively new to e-field modelling. I would really appreciate any guidance with using SimNIBS e-field modelling with Brainsight.

  Thank you!

**Comment by oulap** [comment]
Hi,

You can first import the coordinates from Brainsight into SimNIBS as explained here:

https://simnibs.github.io/simnibs/build/html/documentation/neuronavigation/brainsight.html

And then run a simulation given the coordinates (without the GUI) as shown here:

https://simnibs.github.io/simnibs/build/html/tutorial/scripting.html

Finally, once you have the simulation output from SimNIBS, you an do ROI analysis on the e-fields like this:

https://simnibs.github.io/simnibs/build/html/tutorial/analysis.html

Hope this helps!
